wp-content/themes/access/controllers/site.php

```diff
@@ -74,6 +74,11 @@ public function addToContext($context) {
       'posts_per_page' => -1
     ));
 
+    /** Get the first alert that is set to site wide */
+    $context['alert_sitewide'] = reset(array_filter($alerts, function($post) {
+      return $post->custom['alert_sitewide'];
+    }));
+
     // /** Only set alerts that have a location */
     // $context['alerts'] = array_filter($alerts, function($post) {
     //   return $post->custom['location'];
```

Comment:
I think you need to update your branch... These changes should already be there.